tests/ci/integration/mariadb_patch/0001-WIP-Avoid-define-template-_template-so-AWS-LC-builds.patch

```diff
@@ -0,0 +1,56 @@
+From 10318f2e70b2cab11537b4fcf10266de254d8452 Mon Sep 17 00:00:00 2001
+From: Otto Kekalainen <ottoke@amazon.com>
+Date: Mon, 28 Nov 2022 20:27:35 -0800
+Subject: [PATCH 1/4] WIP: Avoid '#define template _template' so AWS-LC builds
+ can run
+
+Disable the WolfSSL specific bug fix as it breaks AWS-LC builds with
+errors like:
```

Comment:
Why is the AWS-LC build enabling WolfSSL bug fixes?

Reply:
Sorry, I don't have a good answer to what MariaDB/WolfSSL was doing under the hood here :( If it helps, I looked into the files and it seems like the WolfSSL bug fix was being performed, regardless of what backend crypto was being used.
* https://github.com/MariaDB/server/blob/11.1/include/violite.h#L150